config/viv_config.h

```diff
@@ -128,6 +128,10 @@ struct viv_libinput_config the_libinput_configs[] = {
         .left_handed = 0,
         // Invert scroll direction
         .natural_scroll = 0,
+        // Disable while typing
```

Comment:
Am I right to understand this is expected to be available only for touchpads? If so, could you tweak the comment to `Disable input while typing (usually touchpad only)` or similar?

Reply:
done